modules/ROOT/pages/_partials/keycloak/setting-up-the-idm-sdk.adoc

```diff
@@ -70,20 +70,20 @@ Cordova::
 +
 --
 // end::excludeDownstream[]
-. Add `* file:` to `default-src` in the `Content-Security-Policy` meta tag inside the `index.html` file:
+. Add the following code to `index.html` located in `src/` :
 +
 [source,html]
 ----
 <meta http-equiv="Content-Security-Policy" content="default-src * file: 'self' data: gap: https://ssl.gstatic.com 'unsafe-eval'; style-src 'self' 'unsafe-inline'; media-src *; img-src 'self' data: content:;">
 ----
-. Install the link:https://www.npmjs.com/package/@aerogear/auth[AeroGear Auth] package from link:https://www.npmjs.com/[NPM^]:
+. Install the link:https://www.npmjs.com/package/@aerogear/auth[AeroGear Auth] package from link:https://www.npmjs.com/[NPM^] in `src/`:
 +
 [source,bash]
 ----
 $ npm install @aerogear/auth
 ----
-. Use e.g `browserify` so that you can use the `require` statement
-. In the `init` snippet use `require` instead of `import`
+. Use `browserify` so that you can use the `require` statement.
```

Comment:
Hi @austincunningham , @ziccardi - would you mind taking a look at this line and the next line, and suggest an enhancement?
For eg, line 85: What is the purpose of using the require statement - why would I want to use it, and if I use it, what does it do?
For eg, line 86: What is the "init snippet", and do I simply replace `import` with `require`?

Reply:
`require` is how to use node modules , node modules don't work in browsers/cordova so you need to bundle them into native JS. That what browserify does.  `Import` is a more modern version of `require` but I would still use `require` with a cordova app.  see the screen shot for `init` 
![image](https://user-images.githubusercontent.com/16667688/56034482-4b49ae00-5d1f-11e9-8f00-38c7eddfbe62.png)